src/content/en/fundamentals/security/credential-management/save-forms.md

```diff
@@ -140,10 +137,14 @@ or proceed to the personalized page.
       e.preventDefault();
 
       // Try sign-in with AJAX
-      fetch(/'signin', {
+      let form = new FormData();
+      form.append('email', c.id);
+      form.append('password', c.password);
+
+      return fetch('/signin', {
         method: 'POST',
-        body: new FormData(e.target),
-        credentials: 'include'
+        credentials: 'include',
+        body: form
       }).then(res => {
```

Comment:
Not sure why I see `let form = new FormData();` etc (line 139 - 142) which I don't see in [the original code](https://github.com/google/WebFundamentals/blob/master/src/content/en/fundamentals/security/credential-management/save-forms.md).

Depending on having it or not, lines 160 and 161 should be changed.